src/index.html

```diff
@@ -7,21 +7,403 @@
+          </li>
+
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__product"
+          src="/src/images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+        />
+
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__stars-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__stars-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price">
+          <strong class="card__price-price">Price:</strong>
+          <span class="card__price-value">$2,199</span>
+        </div>
+        <a
+          class="card__botton"
+          href="#"
+          data-qa="card-hover"
+        >
+          <div>Buy</div>
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__product"
+          src="/src/images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+        />
+
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__stars-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__stars-reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price">
+          <strong class="card__price-price">Price:</strong>
+          <span class="card__price-value">$2,199</span>
+        </div>
+        <a
+          class="card__botton"
+          href="#"
+        >
+          <div>Buy</div>
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```